requirements.txt

```diff
@@ -1,62 +1,56 @@
 Django==2.1.8
-Pillow==4.3.0                   # PIL fork (Python Imaging Library)
-djangorestframework==3.7.7      # API v1
+Pillow==6.0.0                   # PIL fork (Python Imaging Library)
+djangorestframework==3.9.4      # API v1
 django-filter==2.1.0           # Filtering for DRF
 # python-memcached==1.57          # Memcache. Used by Mailinglists to fetch from Sympa.
 # Upstream is missing Python 3 patches
 git+https://github.com/JelteF/python-memcached@patch-1#egg=python-memcached==1.58
 markdown2==2.3.7                # textarea text formatting
 pytz                            # Timezone lib
-stripe==2.27.0                  # Stripe payment
-icalendar==4.0.1                # iCalendar generation
-google-api-python-client==1.6.5
+stripe==2.28.1                  # Stripe payment
+icalendar==4.0.3                # iCalendar generation
+google-api-python-client==1.7.9
 celery==4.3.0                   # Async task runner
 
 # third party deps
-django-filebrowser==3.10.1      # File uploading
+django-filebrowser==3.11.1      # File uploading
 git+https://github.com/dotKom/django-chunks.git@e6bf109d4fd286964024c140ac1b8f3d86713540#egg=django-chunks==0.4
 django-crispy-forms==1.7.2      # nice forms
-django-extensions==1.9.7        # nice extra commands for debugging, etc
+django-extensions==2.1.7        # nice extra commands for debugging, etc
 django-dynamic-fixture==2.0.0   # Dynamic fixtures for models
-django-recaptcha==1.4.0         # Google ReCaptcha
-django-oauth-toolkit==1.0.0    # OAuth2 authentication support
+django-recaptcha==2.0.4         # Google ReCaptcha
+django-oauth-toolkit==1.2.0    # OAuth2 authentication support
 django-watson==1.5.2            # Indexed model search lib
-django-reversion==2.0.13        # Model version history with middleware hooks to all post_save signals
-django-guardian==1.5.0          # Per Object permissions framework
-django-taggit==0.23.0           # Generic multi-model tagging library
-django-taggit-serializer==0.1.5 # REST Framework serializers for Django-taggit
-APScheduler==3.5.1              # Scheduler
+django-reversion==3.0.4        # Model version history with middleware hooks to all post_save signals
```

Comment:
I think it would be wise to be careful here.

From what I can see we've had problems with this before.
We cant see the version history of anything older than 2016 because of a change in this.